R/page.R

```diff
@@ -96,20 +186,26 @@ validateCssPadding <- function(padding = NULL) {
 #'
 #' Create a dashboard layout with a full-bleed header (`title`) and [sidebar()].
 #'
-#' @inheritParams layout_sidebar
-#' @inheritParams page_fillable
 #' @param ... UI elements to display in the 'main' content area (i.e., next to
 #'   the `sidebar`). These arguments are passed to `layout_sidebar()`, which has
 #'   more details.
 #' @param title A string, number, or [htmltools::tag()] child to display as the
 #'   title (just above the `sidebar`).
+#' @inheritParams layout_sidebar
+#' @inheritParams page_fillable
+#' @inheritParams page_navbar
 #'
 #' @export
+#' @family Dashboard page layouts
+#'
 #' @seealso [layout_sidebar()] for 'floating' sidebar layouts.
```

Comment:
This kind of feels like a distraction for `page_sidebar()` (expecially given there is a `sidebar` argument that points to the same page)

```suggestion
```